scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,68 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
+pkgrel=1
+pkgdesc="The Solidity Contract-Oriented Programming Language (Including solc, lllc; from latest unstable git version)"
+arch=('i686' 'x86_64')
+depends=(
+  'boost'
+  'boost-libs'
+  'crypto++'
+  'jsoncpp'
+  'glibc'
+)
+makedepends=(
+  'autoconf'
+  'automake'
+  'cmake'
+  'gcc'
+  'gcc-libs'
+  'libtool'
+  'yasm'
```

Comment:
I would like to review the package building here since I am also an Arch user. So a question for you. I can build solidity fine from source normally.

But I don't have `yasm`. What is this for here?